doc/source/whatsnew/v1.3.3.rst

```diff
@@ -21,6 +21,15 @@ Fixed regressions
 
 .. ---------------------------------------------------------------------------
 
+.. _whatsnew_133.performance:
+
+Performance improvements
+~~~~~~~~~~~~~~~~~~~~~~~~
+- Performance improvement for :meth:`DataFrame.__setitem__`
```

Comment:
Could you clarify in which cases?

Reply:
Hi @phofl - updated. Let me know if this is fine.